spec/controllers/auth_key_pair_cloud_controller_spec.rb

```diff
@@ -1,61 +1,52 @@
 describe AuthKeyPairCloudController do
-  context "#button" do
-    before(:each) do
-      stub_user(:features => :all)
-      EvmSpecHelper.create_guid_miq_server_zone
-    end
+  let(:kp) { FactoryGirl.create(:auth_key_pair_cloud, :name => "auth-key-pair-cloud-01") }
+  let(:classification) { FactoryGirl.create(:classification, :name => "department", :description => "Department") }
+  let(:tag1) { FactoryGirl.create(:classification_tag, :name   => "tag1", :parent => classification) }
+  let(:tag2) { FactoryGirl.create(:classification_tag, :name   => "tag2", :parent => classification) }
+  let!(:user) { stub_user(:features => :all) }
```

Comment:
Do you actually reference the `user` somewhere below? If not, please move this into the `before` block.